src/openforms/submissions/tests/test_tasks_pdf.py

```diff
@@ -243,4 +245,74 @@ def test_cosign_data_included(self):
             """,
             key=_("Co-signed by"),
         )
-        self.assertInHTML(expected, rendered, count=1)
+        self.assertInHTML(expected_representation, rendered, count=1)
+
+        # check identifier
+        expected_identifier = format_html(
+            """
+            <div class="submission-step-row">
+                <div class="submission-step-row__label">{key}</div>
+                <div class="submission-step-row__value">123456782</div>
+            </div>
+            """,
+            key=_("Identifier of co-signer"),
+        )
+        self.assertInHTML(expected_identifier, rendered, count=1)
+
+    def test_incomplete_cosign_data(self):
```

Comment:
I would split this into two tests... But first check with @joeribekker or @LaurensBurger if this is what the user story describes. I read the ticket as: require a BSN and show it + name if we know it.